src/routes/github/github-router.ts

```diff
@@ -25,36 +39,35 @@ subRouter.post("/webhooks",
 	returnOnValidationError,
 	WebhookReceiverPost);
 
-//Have an cover all middleware to extract the optional gitHubAppId
-//subRouter.use(param("uuid").isUUID('all'), GithubServerAppMiddleware);
+// Create-branch is seperated above since it currently relies on query param to extract the jirahost
+subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, GithubCreateBranchRouter);
 
-// TODO: use jiraSymmetricJwtMiddleware and GithubServerAppMiddleware for all paths below once "create branch" supports JWT
-// subRouter.use(jiraSymmetricJwtMiddleware);
-// subRouter.use(GithubServerAppMiddleware);
+// OAuth Routes
+subRouter.use(GithubOAuthRouter);
```

Comment:
Remove this line? It is duplicated.

Reply:
Removed the other one, and will keep the middleware on the router directly since its a mixed bag of needing some